TestCalc.cs

```diff
@@ -0,0 +1,69 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class TestCalc
+    {
+        [Fact]
+        public void Test()
+        {
+            Assert.True(true);
+        }
+
+        [Fact]

```

Comment:
Please - use InlineData to check multiple cases

Reply:
Exampl:
![image](https://user-images.githubusercontent.com/71152598/144902397-132be493-1b20-4a61-8f32-598a0d94cb51.png)
![image](https://user-images.githubusercontent.com/71152598/144902451-606c147f-9276-4d1e-92c1-98c8a8acb4ca.png)